src/js/util.js

```diff
@@ -206,6 +206,18 @@ export async function fetchText(url, opts) {
 
 /** @this {Object} DriveOptions */
 export function fetchWebDAV(url, init = {}) {
+  // SECURITY: Enforce HTTPS for remote WebDAV URLs to prevent credential exposure
```

Comment:
I'm not sure we can enforce https because many people may still use legacy servers on intranet or private home networks.